### PR TITLE
style: Update filenames to vue.js style

### LINKED ISSUE #660: [Vue] Cleanup css file names
- [ ] use vue filenames

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/64162737-vue-cleanup-css-file-names?utm_campaign=plugin&utm_content=tracker%2F46751899&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F46751899&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by SerekKiri**
I think this should solve this issue 👍 
Fix #660

### CODE REVIEW

File: css/contacts.scss [hunk 1]
```diff
@@ -38,12 +38,12 @@ $grid-input-height-with-margin: #{$grid-height-unit - $grid-input-margin * 2};
 }
 
 
-@import 'settings-section';
-@import 'settings/settings-addressbook';
-@import 'settings/settings-addressbook-shares';
-@import 'settings/settings-addressbook-sharee';
+@import 'SettingsSection';
+@import 'settings/SettingsSddressbook';
```
Comment (skjnldsv) [inline css/contacts.scss]: typo :)

Comment (skjnldsv) [inline css/contacts.scss]: typo :)

Comment (skjnldsv) [inline css/contacts.scss]: Capital S to settings ;)

Comment (SerekKiri) [reply css/contacts.scss]: 😄 Oh, sorry 💯

**Comment by skjnldsv** [comment]
@SerekKiri I'll let you fix the two issues and we can merge this :)

File: css/contacts.scss [hunk 2]
```diff
@@ -38,12 +38,12 @@ $grid-input-height-with-margin: #{$grid-height-unit - $grid-input-margin * 2};
 }
 
 
-@import 'settings-section';
-@import 'settings/settings-addressbook';
-@import 'settings/settings-addressbook-shares';
-@import 'settings/settings-addressbook-sharee';
+@import 'SettingsSection';
+@import 'Settings/SettingsSddressbook';
```
Comment (skjnldsv) [inline css/contacts.scss]: Still this typo ;)

Comment (SerekKiri) [reply css/contacts.scss]: You mean this one? 
```
@import 'SettingsSection';
@import 'Settings/SettingsSddressbook';
@import 'Settings/SettingsAddressbookShares';
@import 'Settings/SettingsAddressbookSharee';```

Comment (SerekKiri) [reply css/contacts.scss]: Or this 
```
"Sharee"
```

**Comment by SerekKiri** [comment]
Oh, never mind sorry i did not see this typo 😄

File: css/contacts.scss (hunk 2 above)
Comment (skjnldsv) [reply css/contacts.scss]: No no, the Sharee is fine! Only the A in Addressbook :)

**Pull request merged**